sessionctx/variable/varsutil.go

```diff
@@ -357,10 +365,10 @@ const (
 
 // TiDBOptEnableClustered converts enable clustered options to ClusteredIndexDefMode.
 func TiDBOptEnableClustered(opt string) ClusteredIndexDefMode {
-	switch {
-	case strings.EqualFold(opt, "ON") || opt == "1":
+	switch opt {
+	case On:
 		return ClusteredIndexDefModeOn
-	case strings.EqualFold(opt, "OFF") || opt == "0":
+	case Off:
```

Comment:
`strings.EqualFold` is not required because validation also performs normalization.